github/GithubRetry.py

```diff
@@ -67,6 +67,12 @@ class GithubRetry(Retry):
     # references the class, not the module (due to re-exporting in github/__init__.py)
     __datetime = datetime
 
+    #: Default values to be used for ``status_forcelist``
+    DEFAULT_STATUS_FORCELIST = list(range(500, 600))
+
+    #: Default methods to be used for ``allowed_methods``
+    DEFAULT_ALLOWED_METHODS = Retry.DEFAULT_ALLOWED_METHODS.union({"GET", "POST"})
```

Comment:
Can we move these next to the existing `DEFAULT_SECONDARY_RATE_WAIT` above?

Reply:
This would result to `GithubRetry.DEFAULT_ALLOWED_METHODS` inheriting `urllib3.Retry.DEFAULT_ALLOWED_METHODS`, which might cause confusion. Is that OK? Or should I have it both as a module and class constant?

Reply:
I don't see a problem with a module constant being based on `urllib3.Retry.DEFAULT_ALLOWED_METHODS`.